package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Offers",
-  "version": "1.0.4",
+  "version": "1.0.5",
   "baseBundleVersion": "0.57.5",
   "private": true,
   "scripts": {
-    "start": "pushd mobile && npm start && popd && pushd web/admin && npm start && popd"
+    "start": "pushd mobile && npm run ios && popd && pushd web/admin && npm start && popd"
```

Comment:
Why this change?

Reply:
Not entirely sure how this change occurred after running upgrade-rn

Reply:
👍 looks like this had undone what upgrade-rn had done.